man/man5/zpool-features.5

```diff
@@ -589,5 +591,51 @@ files.
 
 .RE
 
+.sp
+.ne 2
+.na
+\fB\fBencryption\fR\fR
+.ad
+.RS 4n
+.TS
+l l .
+GUID	com.datto:encryption
+READ\-ONLY COMPATIBLE	no
+DEPENDENCIES	extensible_dataset
+.TE
+
+This feature enables the creation of natively encrypted datasets. ZFS will
+encrypt all user data including file and zvol data, file attributes, ACLs,
+permission bits, directory listings, and FUID mappings. ZFS will not encrypt
+metadata related to the pool structure, including dataset names, dataset
+hierarchy, file size, file holes, and dedup tables. Key rotation is managed
+internally by the ZFS kernel module and changing the user's key does not
+require re-encrypting the entire dataset. Datasets can be scrubbed, resilvered,
+moved, and deleted without the encryption keys being loaded (see the zfs key
+subcommand for more info).
+
+Encryption changes the behavior of a few ZFS operations. Encryption is applied
+after compression so compression ratios are preserved. Normally checksums in
+ZFS are 256 bits long, but for encrypted data the checksum is 128 bits of the
+user-chosen checksum and 128 bits of MAC from the encryption suite, which
+provides additional protection against maliciously altered data. Deduplication
+is still possible with encryption enabled but for security, datasets will only
+dedup against themselves, their snapshots, and their clones. Encrypted data
+cannot be embedded via the embedded data feature.
```

Comment:
I think this should be embedded_data with the underscore, to match the feature flag name.